Tests/Benchmarks/QuickMathBenchmark.cpp

```diff
@@ -0,0 +1,90 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/data/test_case.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Tests/CommonHelpers/BenchmarkTools.hpp"
+#include "Acts/Utilities/QuickMath.hpp"
+
+#include <cmath>
+#include <random>
+
+namespace bdata = boost::unit_test::data;
+
+namespace Acts::Test {
+
+/// @brief Another fast power function @see `fastPow`
+// Taken from
+// https://martin.ankerl.com/2007/02/11/optimized-exponential-functions-for-java
+/// @param a the base
+/// @param b the exponent
+constexpr double fastPowAnother(double a, double b) {
+  // enable only on IEEE 754
+  static_assert(std::numeric_limits<double>::is_iec559);
+
+  union {
+    double f;
+    std::int64_t i;
+  } u = {};
+
+  u.i = static_cast<std::int64_t>(
+      9076650 * (a - 1) / (a + 1 + 4 * std::sqrt(a)) * b + 1072632447);
+  u.i <<= 32;
+
+  // result seems broken?
+  return u.f;
+}
+
+// Some randomness & number crunching
+unsigned int ntests = 10;
+unsigned int nrepts = 10000;
```

Comment:
```suggestion
const unsigned int nTests = 10;
const unsigned int nReps = 10000;
```

Reply:
Since this is only testing / benchmarking code I would not care too much. I actually copied this over from another benchmark.